yaksh/templates/yaksh/question.html

```diff
@@ -199,14 +199,17 @@
          {% else %}
          <button class="btn btn-primary" type="submit" name="check" id="check" onClick="submitCode();">Check Answer <span class="glyphicon glyphicon-cog"></span></button>&nbsp;&nbsp;
          {% endif %}
+         {% if paper.question_paper.quiz.can_skip_questions %}
          {% if paper.unanswered.all|length != 1 %}
```

Comment:
This template value of `paper.unanswered.all` (I think it should be `questions_unanswered`) is probably wrong, please check this and bundle that check into the same, no need for two nested ifs here.